pmm-app/src/pmm-qan-app-panel/QueryAnalytics/DataOutput/QueryDetails/Example/Example.service.ts

```diff
@@ -0,0 +1,21 @@
+class ExampleService {
+  static async getExample({ filterBy, groupBy, labels, periodStartFrom, periodStartTo, tables }) {
+    const data = {
+      filter_by: filterBy,
+      group_by: groupBy,
+      labels: labels || [],
+      period_start_from: periodStartFrom,
+      period_start_to: periodStartTo,
+      tables: tables || [],
+    };
+
+    const response = await fetch('/v0/qan/ObjectDetails/GetQueryExample', {
+      method: 'POST',
+      body: JSON.stringify(data),
+    });
+
+    return response.json();
```

Comment:
should ideally be `return await response.json()`, since `.json` is another promise